### PR TITLE
doc: Translate from Chinese into English for several files for their comments

### PR DESCRIPTION
**Pull request by ghost**
Because there are still many comments in Chinese instead of in English, we must translate them into English in union.

### CODE REVIEW

**Comment by ghost** [comment]
/cc：@popomore

In Nodejs v10.x, there seems to be a CI error, and everything goes well with me solely on my machines.

File: lib/helper/shtml.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const utils = require('../utils');
 
 // judge whether in domain white list or not
 function isSafeDomainCustom(whitelist, domain) {
-  // plus . , because whitelist startwith .
+  // plus with `.` , because `whitelist` starts with `.`
```
Comment (SunShinewyf) [inline lib/helper/shtml.js]: 这里建议使用 add `.`, plus 感觉是一种运算

File: lib/helper/shtml.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ module.exports = function shtml(val) {
       }
     };
 
-    // avoid override user configuration 'onTagAttr'
+    // avoid overridding user configuration 'onTagAttr'
```
Comment (SunShinewyf) [inline lib/helper/shtml.js]: 单词拼写：overridding->overriding

File: lib/middlewares/csp.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ module.exports = options => {
       value = matchedOption[key];
       value = Array.isArray(value) ? value : [ value ];
 
-      // sandbox 单独处理，其他数组拆开为字符串
+      // Other arrays are splitted into strings EXCEPT `sandbox`
```
Comment (SunShinewyf) [inline lib/middlewares/csp.js:41]: are splitted into -> are split into

File: lib/middlewares/hsts.js [hunk 4]
```diff
@@ -11,7 +11,8 @@ module.exports = options => {
     if (utils.checkIfIgnore(opts, ctx)) return;
 
     let val = 'max-age=' + opts.maxAge;
-    // 如果这个可选的参数定义了，这条规则对于网站的所有子域同样生效。
+    // If `opts.includeSubdomains` has value,
+    // the rule will also apply on all the sub domains of the website
```
Comment (SunShinewyf) [inline lib/middlewares/hsts.js]: 建议改成：If  `opts.includeSubdomains` is defined, the rule is also valid for all the sub domains of the website

File: lib/safe_redirect.js [hunk 5]
```diff
@@ -6,11 +6,13 @@ const delegate = require('delegates');
 module.exports = app => {
 
   /**
-   * 不安全的跳转，跳转过程中不检查目标 url 是否安全。
-   * 非特殊情况下，请不要随意使用此接口，否则可能会出现安全漏洞。
+   * This is an unsafe redirection, and we WON'T check if the
+   * destination url is safe or not.
+   * Please DO NOT use this method unless in some very special cases,
+   * otherwise you are running the risk of security vulnerabilities.
    *
```
Comment (SunShinewyf) [inline lib/safe_redirect.js]: 最后一句感觉不太好：否则可能会出现安全漏洞 => otherwise there may be security vulnerabilities，表示一种可能性

File: lib/safe_redirect.js [hunk 6]
```diff
@@ -36,12 +40,12 @@ module.exports = app => {
   app.response.redirect = function redirect(url, alt) {
     url = (url || '/').trim();
 
-    // 处理 // 情况
+    // Process with double `/`
     if (url[0] === '/' && url[1] === '/') {
```
Comment (SunShinewyf) [inline lib/safe_redirect.js]: 这里建议不要翻译 //, 直接使用 `//`

File: lib/safe_redirect.js [hunk 7]
```diff
@@ -65,7 +69,7 @@ module.exports = app => {
             this.ctx.coreLogger.warn('[egg-security:redirect] %s', message);
             url = '/';
           } else {
-            // 非生产环境，直接抛异常提示
+            // Exception will be thrown out for a non-PROD env.
```
Comment (SunShinewyf) [inline lib/safe_redirect.js]: for 建议 改成 in

File: lib/safe_redirect.js [hunk 8]
```diff
@@ -36,12 +40,12 @@ module.exports = app => {
...
     url = (url || '/').trim();
 
-    // 处理 // 情况
+    // Process with double `/`
     if (url[0] === '/' && url[1] === '/') {
       url = '/';
     }
 
-    // 如果以 / 开头，说明系统内跳转
+    // An internal rediction, starting with `/`
```
Comment (SunShinewyf) [inline lib/safe_redirect.js]: 这个感觉没有翻译出来意思，个人建议直译：if begin with '/', it means an internal jump

File: lib/safe_redirect.js (hunk 7 above)
Comment (ghost) [reply lib/safe_redirect.js]: Fixed

File: lib/safe_redirect.js (hunk 8 above)
Comment (ghost) [reply lib/safe_redirect.js]: Fixed

File: lib/safe_redirect.js (hunk 6 above)
Comment (ghost) [reply lib/safe_redirect.js]: Fixed

File: lib/safe_redirect.js (hunk 5 above)
Comment (ghost) [reply lib/safe_redirect.js]: Fixed

File: lib/middlewares/hsts.js (hunk 4 above)
Comment (ghost) [reply lib/middlewares/hsts.js]: Fixed

File: lib/middlewares/csp.js (hunk 3 above)
Comment (ghost) [reply lib/middlewares/csp.js:41]: Why? `split` is a verb, this should use a passive voice.

File: lib/helper/shtml.js (hunk 2 above)
Comment (ghost) [reply lib/helper/shtml.js]: Fixed

File: lib/helper/shtml.js (hunk 1 above)
Comment (ghost) [reply lib/helper/shtml.js]: Fixed

**Review by fengmk2 — APPROVED** [review]

File: lib/middlewares/csp.js (hunk 3 above)
Comment (SunShinewyf) [reply lib/middlewares/csp.js:41]: Because `split` 's passive voice is `split`, not `splitted`

**Pull request merged**